pkg/proxy/iptables/proxier_test.go

```diff
@@ -2330,15 +2330,15 @@ COMMIT
 -A KUBE-MARK-MASQ -j MARK --set-xmark 
 -A KUBE-SERVICES -m comment --comment "ns1/svc1: cluster IP" -m  -p  -d 172.20.1.1/32 --dport 0 ! -s 10.0.0.0/24 -j KUBE-MARK-MASQ
 -A KUBE-SERVICES -m comment --comment "ns1/svc1: cluster IP" -m  -p  -d 172.20.1.1/32 --dport 0 -j KUBE-SVC-3WUAALNGPYZZAWAD
--A KUBE-SVC-3WUAALNGPYZZAWAD -m statistic --mode random --probability 0.33333 -j 
--A  -s 10.0.1.1/32 -j KUBE-MARK-MASQ
--A  -m  -p  -j DNAT --to-destination 10.0.1.1:80
--A KUBE-SVC-3WUAALNGPYZZAWAD -m statistic --mode random --probability 0.50000 -j 
--A  -s 10.0.1.2/32 -j KUBE-MARK-MASQ
--A  -m  -p  -j DNAT --to-destination 10.0.1.2:80
--A KUBE-SVC-3WUAALNGPYZZAWAD -j 
--A  -s 10.0.1.3/32 -j KUBE-MARK-MASQ
--A  -m  -p  -j DNAT --to-destination 10.0.1.3:80
+-A KUBE-SVC-3WUAALNGPYZZAWAD -m comment --comment ns1/svc1: -m statistic --mode random --probability 0.33333 -j 
```

Comment:
As mentioned before, it makes sense that they comment sections indeed bloat the output.

Reply:
The comment in the code there says "So if total number of endpoint chains is large enough, we simply drop those comments" and this small test doesn't exceed the threshold for not adding comments.

The code for adding comments was broken though, as described in https://github.com/kubernetes/kubernetes/pull/81886#discussion_r328288746.